### PR TITLE
Definition for nerd

### LINKED ISSUE #239: Define nerd
Submitting for consideration the following term and definition draft from the [DDH](https://medium.com/dictionary-of-digital-humanities):

**Nerd**
An overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it.

### PR DESCRIPTION
**Pull request by arielsilvestri**
Provides definition for "nerd", per #239 .

I think I'm being a bit negative w/ the definition, as there are possible appropriate usages of the word. I figured I reached a good stopping point to collect feedback on what I have so far, and open to discussion with maintainers on whether or not anything else is needed.

### CODE REVIEW

File: 11ty/definitions/nerd.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - fan
+    - aficionado
+    - enthusiast
+  relating to skill or intelligence:
+    - expert
+    - adept
+    - specialist
+---
+
+From the [DHH](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e): "An overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."
```
Comment (tatianamac) [inline 11ty/definitions/nerd.md]:
```suggestion
"an overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."
–[Dictionary of Digital Humanities](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e)
```

**Comment by tatianamac** [comment]
@silvestrijonathan I think this is a strong start! I agree we could add that there is a positive side to nerd which it can have a take-back effect if someone self-identifies in that way. In general I think that the "take-back" idea is an important one to the core of this dictionary.

I also wonder if we can open up an undefined term of Blerd (Black nerd) so that someone who identifies as a Black nerd could define that, and we can cross-reference them. 🤔

Let us know if you feel comfortable addressing that edit. If not, I'm happy to vet to see if someone else would like to jump on to do some writing. 🤓

**Comment by arielsilvestri** [comment]
@tatianamac I can make some time to add a `take-back` section! I have some thoughts swimming in my head. Let me get some ideas down on paper and see how it hashes out.

Agreed for Blerd! I think holding space and being specific for that is a great idea.

**Comment by arielsilvestri** [comment]
Added a "Taking it back" section, as well as an empty page for blerd and a link to it in this posting.

File: 11ty/definitions/blerd.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Blerd (Black Nerd)
```
Comment (tatianamac) [inline 11ty/definitions/blerd.md]:
```suggestion
title: Blerd
```

File: 11ty/definitions/nerd.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+slug: nerd
+flag:
+  level: avoid
+defined: true
+speech: noun
+alt_words:
+  relating to passion:
+    - fan
+    - aficionado
+    - enthusiast
```
Comment (tatianamac) [inline 11ty/definitions/nerd.md]:
```suggestion
    - aficionado
    - enthusiast
    - fan
```

File: 11ty/definitions/nerd.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  relating to skill or intelligence:
+    - expert
+    - adept
+    - specialist
+---
```
Comment (tatianamac) [inline 11ty/definitions/nerd.md]:
```suggestion
  relating to skill or intelligence:
    - adept
    - expert
    - specialist
---
```

File: 11ty/definitions/nerd.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+"an overly intellectual person, usually obsessive or lacking social skills, focusing their interests on non-mainstream activities, generally highly technical or related to fiction or fantasy narratives. It is used as a stereotype for either derogatory or praising purposes, depending on who's wielding it."
+–[Dictionary of Digital Humanities](https://medium.com/dictionary-of-digital-humanities/nerd-b7d61b6f536e)
+
+## Issues
+
+The term can be used in a way that leads to weaponizing a person's passion against them, which may manifest as bullying of that person's appearance or background.
```
Comment (tatianamac) [inline 11ty/definitions/nerd.md]:
```suggestion
When used in reference to someone else without their consent, the term can weaponise a person's passion against them, or bully someone based on their appearance or background.
```

File: 11ty/definitions/nerd.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Impact
+
+By assigning negativity to the passion that someone exhibits for the things they enjoy, we invalidate their individuality and reduce the enjoyment they derive from their passions. As a result, we may cause these people to abandon hobbies or interests, instilling a feeling of wrongness in finding joy how they see fit (provided their methods of finding joy are not harmful).
+
+## Taking It Back
+
+Individuals can utilize this term in a self-descriptive way to identify that they are extremely passionate about their hobbies and interests. Acknowledging this usage and supporting people who see themselves this way is an effective method of reclaiming the positive aspects of this term and validating the way people see and think of their passions as it relates to their identity.
```
Comment (tatianamac) [inline 11ty/definitions/nerd.md]:
```suggestion
## Self-Identification

Individuals may use this term to self-identify and to indicate their extreme passionate for hobbies and interests. Acknowledging this usage and supporting people who self-identify and reclaim this term can validate the way people perceive their passions as it relates to their identity.
```

Comment (tatianamac) [reply 11ty/definitions/nerd.md]: I'd additionally recommend moving this up immediately before Issues

**Review by tatianamac — APPROVED** [review]
Suggested changes then it's good to go! 🚀

File: 11ty/definitions/nerd.md (hunk 6 above)
Comment (arielsilvestri) [reply 11ty/definitions/nerd.md]: Can do!

**Comment by arielsilvestri** [comment]
@tatianamac Alright, we're all squared away!

Thank you for the grammatical edits, Tatiana. I will make sure to keep things better alphabetized in the future, as well as ensuring to use the preferred English spellings for words. Also a lot of very valuable feedback in here on how I can better construct my thoughts and sentences.

**Comment by arielsilvestri** [comment]
@tatianamac Just a thought - by adding the self-identification section, should we remove the `level: avoid` flag? It may send a mixed message.

**Comment by tatianamac** [comment]
No worries, Jonathan! It's not documented anywhere, so you didn't have any way of knowing! (Working on that 😅)

Yes, good note about the avoid flag: we're addressing the avoid flag issue as a separate holistic PR (#315) that will handle it here as well with some level of urgency. It will help to move flags closer to their framing context, which this definition is a perfect example of!

**Comment by arielsilvestri** [comment]
@tatianamac Sounds good!

**Pull request merged**